examples/reference/panes/HoloViews.ipynb

```diff
@@ -139,24 +414,91 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "pn.panel(dmap, center=True, widget_location='right_bottom')"
+    "print(dmap_panel)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Lets try to change the name of the `FloatSlider`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "float_slider = hv_panel[1][0]\n",
+    "float_slider.name=\"Frequency (updated)\"\n",
+    "float_slider"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets try to reorganize the layout"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "widgets = hv_panel[1]\n",
+    "\n",
+    "pn.Row(pn.Column(*widgets), hv_panel[0])"
```

Comment:
When I reorganize the layout it raises an exception.

![image](https://github.com/holoviz/panel/assets/42288570/b1932da7-3f05-4790-be67-d399de49abc9)